guides/third_party_cdns.md

```diff
@@ -0,0 +1,25 @@
+# Third Party CDNs
+
+Moovweb XDN is designed and built to be the component of your site that your users will directly connect to from their devices. Such components are colloquially known as "edge components". But sometimes you may prefer, usually not due to technical reasons, to run XDN behind a third-party CDN. XDN fully supports this use case but it's important to call out some common pitfals with this kind of network topology.
+
+## Split Testing
+
+Moovweb XDN offers fully featured [split testing](/guides/split_testing) but when running behind another CDN, you must configure that CDN in a very [specific way](https://developer.moovweb.com/guides/split_testing#section_third_party_cdns)
+
+## IPs
+
+When behind a third-party CDN, Moovweb XDN will analyze `x-forwarded-for` to correctly extract the client IP from it and inject it into `x-xdn-client-ip`. You can continue to use `x-xdn-client-ip` as you otherwise would.
```

Comment:
```suggestion
When behind a third-party CDN, Moovweb XDN will analyze `x-forwarded-for` to correctly extract the client IP from it and inject it into [`x-xdn-client-ip`](request_headers#section_general_headers). You can continue to use `x-xdn-client-ip` as you otherwise would.
```